Sources/MapboxNavigation/SpeedLimitView.swift

```diff
@@ -1,6 +1,14 @@
 import UIKit
 import MapboxDirections
 
+/**
+ Indicates which view mode to display, the current speed or the speed limit.
+ */
+enum SpeedViewMode {
```

Comment:
I wonder if we should really expose such enum at this point. What if we just show current speed if its present and only show speed limit if current speed is not available?

Reply:
Because right now it's not public, and I'm not sure how the `SpeedLimitView` should be used. We could remove it in the design PR.